Add tip about unused docker volumes cleaning

asaunier opened the pull request:
This PR adds instructions to remove "ghost" volumes, as described at https://techan.fr/docker-chasse-a-lespace-perdu-gare-aux-volumes.html > "Les volumes “fantômes”", pointed by @yjacolin 

By the way I have dropped a reference to ExtJS in the documentation opening page :P 

(Doc successfully built on my laptop)

sbrunner commented on doc/integrator/docker.rst:
```diff
@@ -72,6 +72,17 @@ This will remove::
 * Unnamed dangling volumes.
 
+To remove unused volumes created by possibly unconsistent volume declarations
+in the ``Dockerfile`` compared to ``docker-compose.yml``:
```
```suggestion
in the ``Dockerfile`` compared to ``docker-compose.yaml``:
```

sbrunner commented on doc/integrator/docker.rst:
```diff
@@ -72,6 +72,17 @@ This will remove::
+    for vol in $(docker volume ls | awk '{print $2}' | grep -v VOLUME)
+    do
+      docker volume rm $vol
+    done
```
Why not using?
```
docker system prune --volumes 
```

asaunier replied:
I have added the tip, as mentioned in https://techan.fr/docker-chasse-a-lespace-perdu-gare-aux-volumes.html and used by @yjacolin for customers that recently had disk space issues. I have no clue if both commands do the same thing.

@yjacolin what do you think? Have customers used the prune command as well?

asaunier replied:
@yjacolin Oops, I should have read your next comment before posting :P

yjacolin commented on doc/integrator/docker.rst:
```diff
@@ -72,6 +72,17 @@ This will remove::
+    do
+      docker volume rm $vol
+    done
+
```
For last release of Docker you can use::
`docker system prune --volumes`
or::
`docker system prune --all`

asaunier commented:
@yjacolin @sbrunner I have updated the PR to use "system prune --volumes".
Or should we mention also the other command (see d7f3764) for older versions of Docker?

sbrunner commented:
For me, we (especially me) are wrong, the old version war correct because here we are speaking about Docker < 1.13 who don't support `Docker system prune`...

asaunier commented:
@sbrunner You're right, thanks!
By the way, I hadn't paid attention to this but the doc was already mentioning ``docker system prune`` for Docker >= 1.13, a few lines above:
See https://github.com/camptocamp/c2cgeoportal/blob/master/doc/integrator/docker.rst#clean

Then the section I am editing is for Docker < 1.13 and it's better to document the old way only, based on the ``for`` loop. I have updated my PR and squashed the previous fixups.

@yjacolin Do you maintain your change request?

@jwkaltz Thanks for your review!

sbrunner commented on doc/integrator/docker.rst:
```diff
@@ -66,12 +66,23 @@ Otherwise:
    docker volume ls --quiet --filter dangling=true | grep '[0-9a-f]\{64\}' | xargs --no-run-if-empty docker volume rm
 
-This will remove::
+This will remove:
```
Are you sure about this change?

asaunier replied:
Yes, else the following bullet points are not interpreted and displayed as a code section.
Here is how the page looks like after my changes:
![Screenshot from 2020-02-17 11-13-38](https://user-images.githubusercontent.com/1192331/74644477-9f5ec380-5176-11ea-898c-ff5c1325b596.png)

sbrunner replied:
+1 thanks :-)

asaunier commented:
@sbrunner Since you have already done some related changes against branch master in PR https://github.com/camptocamp/c2cgeoportal/pull/5832 I have updated the current PR to merge to branch 2.4 instead of master.

asaunier commented:
Jenkins has finally passed. I will merge the PR soon if no one objects in the meantime.
